cmd/release-controller/http.go

```diff
@@ -288,6 +288,7 @@ func (c *Controller) userInterfaceHandler() http.Handler {
 	mux := mux.NewRouter()
 	mux.HandleFunc("/graph", c.graphHandler)
 	mux.HandleFunc("/changelog", c.httpReleaseChangelog)
+	mux.HandleFunc("/api/v1/releaseInfo", c.httpReleaseInfoJson)
```

Comment:
We haven't really declared the current release output as part of a stable api, so this should be `/releasetag/{tag}/json`